packages/client/hmi-client/src/components/model/petrinet/tera-parameter-entry.vue

```diff
@@ -42,12 +42,12 @@
 						:options="distributionTypeOptions()"
 						class="mr-3"
 					/>
-
 					<!-- Constant -->
 					<tera-input-number
 						v-if="getParameterDistribution(modelConfiguration, parameterId).type === DistributionType.Constant"
 						label="Constant"
 						:model-value="getParameterDistribution(modelConfiguration, parameterId)?.parameters.value"
+						:error-empty="errorEmpty"
```

Comment:
ya I'd just assign them to true here it'll always be like that. 

Side tip: you can put `error-empty` as a short form for`:error-empty="true"`